### ISSUE TITLE
Pollen Count?

### ISSUE THREAD

**Issue by chris-gillatt** [chubin/wttr.in#749]
Is there a way to show Pollen count on wttr? Either via a flag or on the main url? I've not been able to find one in the docs or code so this may well be a feature request. 

Pollen counts are useful to those who suffer with hay fever and other respiratory illnesses, so would be useful to many.

Here's an example of the data required, it mainly comes down to one or two characters to render, L, M, H, VH:

https://www.metoffice.gov.uk/weather/warnings-and-advice/seasonal-advice/pollen-forecast#?date=2022-06-08&region=nw

**Comment by chris-mb** [comment]
I don't think this is possible with the current [WeatherOnline data source](https://www.worldweatheronline.com/developer/api/local-city-town-weather-api.aspx) , would be a good one to bring in once other data sources are introduced.

**Comment by chubin** [comment]
It possible to combine different source together; there are a lot of other interesting sources that would be great to add.
Would you like to have it for the one-line output format, or for the main view (v1), or for v2 too?
